src/Components/Wizard/types.tsx

```diff
@@ -0,0 +1,21 @@
+import { FormikHandler } from "../Forms/support"
+
+export interface RenderProps {
+  validate?: () => any | FormikHandler
+  next: (v: any) => void
+  previous: (v: any) => void
+  onComplete: (v: any) => void
+  pageIndex: number
+  pages?: WizardSchema
+  values?: any
+}
```

Comment:
The standard bag of props handed down to a wizard's child. Note that I'm not using the handleSubmit method I created in Wizard, leaving it up to the children to decide when to `next`/`onComplete`

*I really think it would be semantically nice to pass the pages in as children to the wizard itself* but I wasn't able to do that at first pass. Open to thoughts on it.